File: app/src/main/java/com/appsbygu/kadai_weather_java/models/apis/Weathers/Location.java

package com.appsbygu.kadai_weather_java.models.apis.Weathers;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import io.realm.RealmObject;

public class Location extends RealmObject {

    @SerializedName("city")
    @Expose
    private String city;
    @SerializedName("area")
    @Expose
    private String area;
    @SerializedName("prefecture")
    @Expose
    private String prefecture;

    public String getCity() {
        return city;
    }

    public String getArea() {
        return area;
    }

    public String getPrefecture() {
        return prefecture;
    }

}

File: app/src/main/java/com/appsbygu/kadai_weather_java/models/apis/weathers/Location.kt
package com.appsbygu.kadai_weather_java.models.apis.weathers

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import io.realm.RealmObject

open class Location : RealmObject() {

    @SerializedName("city")
    @Expose
    var city: String? = null
    @SerializedName("area")
    @Expose
    var area: String? = null
    @SerializedName("prefecture")
    @Expose
    var prefecture: String? = null

}
